AudioKit/Core Classes/AKManager.h

```diff
@@ -29,6 +29,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Determines whether or not to log
 @property BOOL isLogging;
 
+/// Place to put lines of testing logs
+@property NSMutableArray *testLog;
+
```

Comment:
OK, then I suggest we bracket this property and the corresponding code in the message delegate so it doesn't even get included in non-test builds (save a bit of memory). That's what that TRAVIS_CI macro is for. :+1: